ui/components/app/assets/nfts/nft-details/__snapshots__/nft-full-image.test.js.snap

```diff
@@ -18,7 +18,7 @@ exports[`NFT full image should match snapshot 1`] = `
           <div
             class="mm-box"
           >
-            <p
+            <div
```

Comment:
We should likely implement logic to conditionally render this element if `children` don’t exist, but we can handle that in a subsequent PR to keep the changes minimal here. An empty `div` makes more sense than an empty `p` tag.